txjuju/_juju1.py

```diff
@@ -48,3 +49,51 @@ def _env_as_dict(self, env):
         if env.bootstrap.admin_secret:
             config["admin-secret"] = env.bootstrap.admin_secret
         return config
+
+
+class CLIHooks(object):
+
+    CFGDIR_ENVVAR = "JUJU_HOME"
+
+    def get_bootstrap_args(self, spec, to=None, cfgfile=None,
+                           verbose=False, gui=False, autoupgrade=False):
+        # Note that for Juju 1.x we ignore gui.
+        if cfgfile is not None:
+            raise ValueError("cfgfile not supported for Juju 1.x bootstrap")
```

Comment:
Right, I suspected something of the sort, but wanted to double check since it looked counter-intuitive. Thanks.